assets/agenda/components/AgendaCoverageExistsFilter.tsx

```diff
@@ -25,6 +26,8 @@ function getActiveFilterLabel(filter: any, activeFilter: any) {
         return gettext('Planned');
     case FILTER_VALUES.NOT_PLANNED:
         return gettext('Not Planned');
+    case FILTER_VALUES.MAY_BE:
+        return gettext('May Be');
```

Comment:
This is the text that appears in the parent button of the dropdown, which has a different label to the option provided in the dropdown. 
I'm wondering if this should rather be `Not Decided` so the option in the dropdown list and the dropdown button text are relatable for end users

Reply:
yep, sounds identical